os/board/rtl8730e/include/board_pins.h

```diff
@@ -0,0 +1,224 @@
+#define         PWR_PIN_57	{_PB_25,		GPIO_PuPd_DOWN,			GPIO_PuPd_DOWN}	// SDA Ext-Flash SPI1_CLK
+#define         PWR_PIN_58	{_PB_26,		GPIO_PuPd_UP,			GPIO_PuPd_UP}	// SDA Ext-Flash SPI1_CS pull up 4.7k
+#define         PWR_PIN_59	{_PB_27,		GPIO_PuPd_UP,			GPIO_PuPd_UP}	// SDA Ext-Flash SPI1_MISO
+#define         PWR_PIN_60	{_PB_28,		GPIO_PuPd_UP,			GPIO_PuPd_UP}	// SDA Ext-Flash SPI0_MOSI
+#define         PWR_PIN_61	{_PB_29,		GPIO_PuPd_NOPULL,		GPIO_PuPd_NOPULL}	// SDA pull up 4.7k
+#define         PWR_PIN_62	{_PB_30,		GPIO_PuPd_NOPULL,		GPIO_PuPd_NOPULL}	// SDA pull up 4.7k
+#define         PWR_PIN_63	{_PB_31,		GPIO_PuPd_DOWN,			GPIO_PuPd_DOWN}	// SDA pull down 4.7k
+#define         PWR_PIN_64	{_PC_0,			GPIO_PuPd_UP,			GPIO_PuPd_UP}	// SDA pull up 4.7k
+#define         PWR_PIN_65	{_PC_1,			GPIO_PuPd_UP,			GPIO_PuPd_UP}	// RTK Int-Flash SPI_FLASH_DATA
+#define         PWR_PIN_66	{_PC_2,			GPIO_PuPd_UP,			GPIO_PuPd_DOWN}	// RTK Int-Flash SPI_FLASH_CLK pull down for PG or 50uA
+#define         PWR_PIN_67	{_PC_3,			GPIO_PuPd_UP,			GPIO_PuPd_UP}	// RTK Int-Flash SPI_FLASH_DATA
+#define         PWR_PIN_68	{_PC_4,			GPIO_PuPd_UP,			GPIO_PuPd_UP}	// RTK Int-Flash SPI_FLASH_DATA
+#define         PWR_PIN_69	{_PC_5,			GPIO_PuPd_UP,			GPIO_PuPd_UP}	// RTK Int-Flash SPI_FLASH_DATA
+#define         PWR_PIN_70	{_PC_6,			GPIO_PuPd_UP,			GPIO_PuPd_UP}	// RTK Int-Flash SPI_FLASH_CS
+#define         PWR_PIN_71	{_PNC,			GPIO_PuPd_KEEP,			GPIO_PuPd_KEEP}	// table end
+
+/* Wakeup pins configuration */
+#define		WAKEPIN_0	((u32)0x00000000)	/*!< see aon_wakepin */
+#define		WAKEPIN_1	((u32)0x00000001)	/*!< see aon_wakepin */
+#define		WAKEPIN_2	((u32)0x00000002)	/*!< see aon_wakepin */
+#define		WAKEPIN_3	((u32)0x00000003)	/*!< see aon_wakepin */
+
+#define IS_UART0_TX(tx) ((tx == PA_3) || (tx == PA_14) || (tx == PA_29) || (tx == PB_6))
+#define IS_UART1_TX(tx) ((tx == PA_5) || (tx == PA_10) || (tx == PA_25) || (tx == PB_11) || (tx == PB_20) || (tx == PB_30))
+#define IS_UART2_TX(tx) ((tx == PA_8) || (tx == PA_12) || (tx == PA_23) || (tx == PB_22))
+#define IS_UART4_TX(tx) (tx == PB_24)
+
+/* I2S2 pins configuration */
+#ifdef CONFIG_AMEBASMART_I2S
+#ifdef CONFIG_AMEBASMART_I2S2
+#define		I2S2_MCLK	NULL
+#define		I2S2_SCLK	PB_21
+#define		I2S2_WS		PA_16
+#define		I2S2_SD_TX	PB_10
+#define		I2S2_SD_RX	PB_19
+#endif							// CONFIG_AMEBASMART_I2S2
+
+#ifdef CONFIG_AMEBASMART_I2S3
+#define		I2S3_MCLK	PA_15
+#define		I2S3_SCLK	PA_14
+#define		I2S3_WS		PA_13
+#define		I2S3_SD_TX	PB_11
+#define		I2S3_SD_RX	PB_20
+#endif							// CONFIG_AMEBASMART_I2S3
+#endif							// CONFIG_AMEBASMART_I2S
+
+/* LCD ST7789 pins configuration */
+#ifdef CONFIG_LCD_ST7789
+#if defined(CONFIG_RTL8730E_BOARD_AIL) || defined(CONFIG_RTL8730E_BOARD_AILP) || defined(CONFIG_RTL8730E_BOARD_AILPW)
+#error "ERROR, Not supported this revision"
+#else
+#define		GPIO_PIN_RESET		PB_11
+#define		GPIO_PIN_BACKLIGHT	PA_9
+#define		GPIO_PIN_CMDDATA	PA_10
```

Comment:
we are consolidating all pin, the some pin label can be duplicated.
please specify device name into each pin label